pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -171,22 +172,22 @@ class VmOverviewTabLibvirt extends React.Component {
         );
         const bootOrder = (
             <div>
```

Comment:
This probably should be a `React.Fragment` (shorthand: `<> </>` for wrapping) instead of a `div`. Same for other empty `div`s below.

Reply:
If I don't wrap these with divs the layout breaks, I was just about to ask you about a proper fix for this.